### PR TITLE
fix: Alert component fixed

### LINKED ISSUE #9572: [CAL-1969] Alert Component not responsive
Found a bug? Please fill out the sections below. 👍

### Issue Summary

I got this banner after creating an organization.

### Steps to Reproduce

1. go to [http://localhost:3000/settings/organizations/new](http://localhost:3000/settings/organizations/new)
2. finish organization onboarding
3. you would be redirected to /event-types with this banner

Alert component in storybook

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1969](https://linear.app/calcom/issue/CAL-1969/alert-component-not-responsive)</sub>

### PR DESCRIPTION
**Pull request by AnitKrJha**
## What does this PR do?

Fixed the responsiveness of  Alert Component , the actions button no longer overlap the content.

### Before
image

### After:

image

Fixes #9572  (issue)

## Type of change

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/prisma/schema.prisma [hunk 1]
```diff
@@ -242,7 +242,7 @@ model User {
 }
 
 model Team {
-  id                  Int                @id @default(autoincrement())
+  id                  Int                     @id @default(autoincrement())
```
Comment (PeerRich) [inline packages/prisma/schema.prisma:245]: can you uncommit your schema.prisma changes?

Comment (PeerRich) [reply packages/prisma/schema.prisma:245]: and uncommit yarn.lock

Comment (AnitKrJha) [reply packages/prisma/schema.prisma]: Okk sure !!

Comment (AnitKrJha) [reply packages/prisma/schema.prisma]: I removed them , review it please

**Review by PeerRich — APPROVED** [review]
great fix! thank you!

File: packages/ui/components/alert/Alert.tsx [hunk 2]
```diff
@@ -71,13 +71,14 @@ export const Alert = forwardRef<HTMLDivElement, AlertProps>((props, ref) => {
...
-
-        <div className="ml-3 flex-grow">
-          <h3 className="text-sm font-medium">{props.title}</h3>
-          <div className="text-sm">{props.message}</div>
+        <div className="flex flex-grow flex-col sm:flex-row">
+          <div className="ml-3 ">
+            <h3 className="text-sm font-medium">{props.title}</h3>
+            <div className="text-sm">{props.message}</div>
+          </div>
+          {/* @TODO: Shouldn't be absolute. This makes it harder to give margin etc. */}
```
Comment (PeerRich) [inline packages/ui/components/alert/Alert.tsx:79]:
```suggestion
```

**Comment by AnitKrJha** [comment]
> great fix! thank you!

It was my first OSS contribution! Feels great ❤️ ! Thanks Peer

**Pull request merged**